maxscale-cdc-adapter/CMakeLists.txt

```diff
@@ -1,11 +1,11 @@
 cmake_minimum_required(VERSION 2.8)
+include_directories(..)
```

Comment:
Same here,  are there headers that the adapter depends on in the source root directory?